python/doc/theory/meta_modeling/cross_validation.rst

```diff
@@ -1,186 +1,654 @@
 .. _cross_validation:
 
-Cross validation assessment of PC models
-----------------------------------------
-
-Once a polynomial response surface :math:`\widehat{h}(\underline{x})`
-of the original numerical model :math:`h(\underline{x})` has been
-built up, it is crucial to estimate the approximation error, i.e. the
+Validation and cross validation of metamodels
+---------------------------------------------
+
+Introduction
+~~~~~~~~~~~~
+
+In ordinary validation of a metamodel, we hold some observations from the
+sample and train the model on the remaining observations.
+Then, we test the metamodel against the hold out observations.
+One of the problems of this method is that the surrogate
+model is trained using only a subset of the available data.
+As a consequence, the estimated error can be pessimistic or, on the contrary, overly optimistic.
+In cross-validation, the entirety of the data set is sequentially used for either training or validation.
+Hence, more of the available data is used, resulting in an improved
+accuracy of the estimated error of the metamodel.
```

Comment:
> You are right: alternating between both words is confusing.
> 
> But "surrogate model" cannot be specific to the French community... because it is not French! So, let's benchmark this.
> 
>     * "Metamodel": 14 languages
> 
>     * "Surrogate model": 3 languages
> 
> 
> So "Metamodel" wins.
> 
> ![image](https://private-user-images.githubusercontent.com/31351465/326035352-adc9af62-37ce-44d0-8ce0-846e5712ecd3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQzMzMzMjksIm5iZiI6MTcxNDMzMzAyOSwicGF0aCI6Ii8zMTM1MTQ2NS8zMjYwMzUzNTItYWRjOWFmNjItMzdjZS00NGQwLThjZTAtODQ2ZTU3MTJlY2QzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI4VDE5MzcwOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTczMzlkNmM1MDU0M2QxMzc1YWI2MTczNWQ1MDBlZTA0YzFmM2ZlY2ZiYTVhNzQ5N2NhMmRjMTViMzAxYjZlMTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.WgcG8LIZYMJjNTAALQyO07Vegco4sfvmZVwAXKamRVs)
> 
> ![image](https://private-user-images.githubusercontent.com/31351465/326035469-4d522760-d2d2-49b1-bb20-98ff80c7c450.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQzMzMzMjksIm5iZiI6MTcxNDMzMzAyOSwicGF0aCI6Ii8zMTM1MTQ2NS8zMjYwMzU0NjktNGQ1MjI3NjAtZDJkMi00OWIxLWJiMjAtOThmZjgwYzdjNDUwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI4VDE5MzcwOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY5ZjgzMGViZTA1MGNlNWYwMDljZGM4YzNiZjlmZGQ5MmFjNGFlMmQ1ZTU5OTY0OTVjODU0Yzg1OWRhOTdkYTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.yXyeDv6Qhkk9m8nAQQ-L-coEiiYRJYVnVJlv1zlCB8Y)

Actually, the Wikipedia article about "metamodeling" is not really about what our community calls a "metamodel"/"surrogate model" but seems to be rather a software engineering concept. The intro of the wikipedia page says:

_Not to be confused with [Surrogate model](https://en.wikipedia.org/wiki/Surrogate_model)._

Things are a little muddled by the "Type of metamodels" section which lists Kriging and neural networks... as software engineering concepts!

Since you mentioned the 14 languages of "Metamodeling" I had a look at the French version. You can see stuff like:

*Dans le cadre d'une [modélisation](https://fr.wikipedia.org/wiki/Mod%C3%A9lisation), un métamodèle est un [modèle](https://fr.wikipedia.org/wiki/Mod%C3%A8le_(informatique)) dont les instances sont elles-mêmes des modèles.*

*Quatre aspects sont nécessaires à la spécification d'un métamodèle : une [syntaxe abstraite](https://fr.wikipedia.org/wiki/Syntaxe_abstraite), une syntaxe concrète, une sémantique statique et une sémantique dynamique[1](https://fr.wikipedia.org/wiki/M%C3%A9tamod%C3%A8le#cite_note-:0-1),[4](https://fr.wikipedia.org/wiki/M%C3%A9tamod%C3%A8le#cite_note-:1-4). Idéalement, ces éléments sont définis à l'aide d'un langage formel[4](https://fr.wikipedia.org/wiki/M%C3%A9tamod%C3%A8le#cite_note-:1-4). Pour cette raison, un métamodèle est aussi parfois défini comme le modèle d'un [langage de modélisation](https://fr.wikipedia.org/wiki/Langage_de_mod%C3%A9lisation)[5](https://fr.wikipedia.org/wiki/M%C3%A9tamod%C3%A8le#cite_note-5).*

On the contrary, when you look at the "surrogate model" page (which has no French equivalent) the intro is:

*A surrogate model is an engineering method used when an outcome of interest cannot be easily measured or computed, so an approximate [mathematical model](https://en.wikipedia.org/wiki/Mathematical_model) of the outcome is used instead.*

My conclusion is that Wikipedia rather supports using "surrogate model" everywhere.

Reply:
Assuming you are right (and it is likely to be so), then we have an issue here, because the word "metamodel" is used throughout the library. For example, the class is named "MetaModelValidation". This is one of the reasons I used this word. If surrogate model is a best choice, then "SurrogateModelValidation" would be best.

Reply:
Yes, this is clearly beyond the scope of this PR, I only posted this for future reference.